doc/whats_new/v0.21.rst

```diff
@@ -116,6 +116,14 @@ Support for Python 3.4 and below has been officially dropped.
   and :class:`tree.ExtraTreeRegressor`.
   :issue:`12300` by :user:`Adrin Jalali <adrinjalali>`.
 
+:mod:`sklearn.utils`
+..................
+- New :func:`utils._init_arpack_v0` which goal is to be used each time eigs,
```

Comment:
This creates a private method. We only need to document user-facing changes in what's new.

Reply:
All right then I'll remove that